tests/models/test_tpu.py

```diff
@@ -200,28 +194,10 @@ def test_model_16bit_tpu_index_5(tmpdir):
     assert os.environ.get('XLA_USE_BF16') == str(1), "XLA_USE_BF16 was not set in environment variables"
 
 
+@pytest.mark.parametrize('tpu_core', [1, 5])
 @pytest.mark.skipif(not TPU_AVAILABLE, reason="test requires TPU machine")
 @pl_multi_process_test
-def test_early_stop_checkpoints_on_tpu(tmpdir):
-    """Test if single TPU core training works"""
-    model = EvalModelTemplate()
-    trainer = Trainer(
-        early_stop_callback=True,
-        default_root_dir=tmpdir,
-        progress_bar_refresh_rate=0,
-        max_epochs=50,
-        limit_train_batches=10,
-        limit_val_batches=10,
-        distributed_backend='tpu',
-        tpu_cores=[1],
-    )
-    trainer.fit(model)
-    assert torch_xla._XLAC._xla_get_default_device() == 'xla:1'
-
-
-@pytest.mark.skipif(not TPU_AVAILABLE, reason="test requires TPU machine")
-@pl_multi_process_test
-def test_early_stop_checkpoints_on_tpu(tmpdir):
+def test_train_on_single_tpu(tmpdir, tpu_core):
```

Comment:
`test_early_stop_checkpoints_on_tpu` was redefined (caught by flake8) and misnamed, so I changed the name of the test and parametrized it.

Reply:
I guess we had some problems with parameterization these types of tests...